Create Conceptual Design.md

ZDElle opened the pull request:
Only words and some pictures. Need to come back and add.

ZDElle committed:
Create Conceptual Design.md

ZDElle committed:
Update Conceptual Design.md

ZDElle committed:
Update Conceptual Design.md

Added images

ZDElle committed:
Update Conceptual Design.md

Adjusted image scaling

csjohnson43 reviewed (changes requested):
Here are my general comments, please be sure to address/rebuttal these in sufficient detail:
•	Eliminate first person usage in the section, Restating the Fully Formulated Problem.
•	When listing a standard, clearly state what metrics in the standard apply to the system? How will they impact the design of the subsystems?
•	In comparative analysis of existing solutions, clearly state the most important metrics for each subsystem. Numbers or cited data should be used to help compare all solutions. Once the metrics have been used to compare potential solutions, then the subsystem solution choice can be stated. A table can be included, like the trade studies we discussed, to clearly show and define solution performance against each other. When listing the potential solutions, you should include a more detailed description of each solution as well.
•	Typo in the User control section of High Level Solutions. (signlas)
•	Text in the hardware block diagram should be increased for easier reading; lines should also be thicker. For every connection, it needs to be clearly labeled what it is. You could make Power connections red and add the data protocol being used to all data lines.
•	Typo in Power and Circuitry subsystem
•	Please be sure to clearly label all inputs and outputs in the atomic subsystem section in sufficient detail. Data signals should list the communication protocol that will be used.
•	Path Finding and Human Detection Specifications should be a list
•	Ethical and Professional Considerations should be more detailed. Some of this information can come from project proposal. 
•	For the standards, make sure to directly link these to the subsystems. I want to see how the subsystem design takes standards into consideration.
•	In the budget section, a clear justification for all parts must be stated. Typically, justification has been given earlier in the document, but you will want to include a brief justification for each item next to it. Some items such as the power and circuitry components can be lumped into one justification. The budget should also show the subtotal for each subsystem, so a total amount of money is budgeted for each subsystem. The total cost of all subsystems must be shown at the end. Is there headway built into this budget so that there is a cushion if a part breaks? While specific parts are not required for this document, make sure that you have considered all possible costs for each subsystem. An example of something that

ZDElle committed:
Update Conceptual Design.md

Fixed typos

ZDElle committed:
Update Conceptual Design.md

Updated for this feedback:

• Ethical and Professional Considerations should be more detailed. Some of this information can come from project proposal.

ZDElle committed:
Update Conceptual Design.md

Updated for this feedback:

• Path Finding and Human Detection Specifications should be a list

ZDElle committed:
Update Conceptual Design.md

Updated for this feedback:

• Eliminate first person usage in the section, Restating the Fully Formulated Problem.

ZDElle committed:
Update Conceptual Design.md

Updated for this feedback:

• In the budget section, a clear justification for all parts must be stated.

ZDElle committed:
Update and rename Reports/Conceptual Design/Conceptual Design.md to Reports/Conceptual Design/Designs/Conceptual Design.md

Updated for this feedback:

• Text in the hardware block diagram should be increased for easier reading; lines should also be thicker. For every connection, it needs to be clearly labeled what it is. You could make Power connections red and add the data protocol being used to all data lines.

ZDElle committed:
Update Conceptual Design.md

Updated for this feedback:

• For the standards, make sure to directly link these to the subsystems. I want to see how the subsystem design takes standards into consideration.

ZDElle committed:
Update Conceptual Design.md

Updated for this feedback:

• When listing a standard, clearly state what metrics in the standard apply to the system? How will they impact the design of the subsystems?

csjohnson43 reviewed (changes requested):
Please address these as soon as possible, you will be approved if these are addressed sufficiently:
- Comparative Analysis of Potential Solutions is still lacking defined metrics with data and citations to reinforce the argument. You need to clearly define the metrics of importance for each solution and state directly using cited data or reasonable arguments how every solution compares. For example, if cost is the most important factor for a solution, you need to state the cost for every potential solution so it is clear why you chose the subsystem solution.
- Please add the total overall price of all subsystems combined to the budget section. Also, are all these items going to be purchased or are there some items that will be inherited from the previous group? If so, I would note that in the budget section.
- On your Gantt chart, please make sure to update the progress column to include either completed, in progress, or planned

ZDElle committed:
Update Conceptual Design.md

Updated for this feedback:

Comparative Analysis of Potential Solutions is still lacking defined metrics with data and citations to reinforce the argument. You need to clearly define the metrics of importance for each solution and state directly using cited data or reasonable arguments how every solution compares. For example, if cost is the most important factor for a solution, you need to state the cost for every potential solution so it is clear why you chose the subsystem solution.
Please add the total overall price of all subsystems combined to the budget section. Also, are all these items going to be purchased or are there some items that will be inherited from the previous group? If so, I would note that in the budget section.
On your Gantt chart, please make sure to update the progress column to include either completed, in progress, or planned

ZDElle committed:
Update Conceptual Design.md

Updated the formatting

ZDElle committed:
Update Conceptual Design.md

Fixed formatting
